src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -13825,8 +13779,6 @@ void VRFCondenserEquipment::VRFOU_CompCap(
     static constexpr std::string_view RoutineName("VRFOU_CompCap");
 
     // variable initializations: component index
-    TUListNum = this->ZoneTUListPtr;
-    NumTUInList = state.dataHVACVarRefFlow->TerminalUnitList(TUListNum).NumTUInList;
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:13829]:(style),[unreadVariable],Variable 'NumTUInList' is assigned a value that is never used.